.shopify-build/krane.yml

```diff
@@ -9,6 +9,14 @@ steps:
       - bundle exec rubocop
     dependencies:
       - bundler
+  - label: 'Run Test Suite (:kubernetes: 1.17-latest :ruby: 2.6)'
+    command: bin/ci
+    agents:
+      queue: k8s-ci
+    env:
+      LOGGING_LEVEL: "4"
+      KUBERNETES_VERSION: v1.17-latest
+      RUBY_VERSION: "2.6"
```

Comment:
How does this work? Isn't the container still `circleci/ruby:2.5.7-node-browsers`?

Reply:
It installs ruby 2.6 https://buildkite.com/shopify/krane/builds/156#7815f427-a96f-4346-9448-4e35686c70a5/209-211